test_runner/fixtures/neon_fixtures.py

```diff
@@ -1575,36 +1576,33 @@ def create_tenant(
             tenant_id = TenantId.generate()
         if timeline_id is None:
             timeline_id = TimelineId.generate()
-        if conf is None:
-            res = self.raw_cli(
-                [
-                    "tenant",
-                    "create",
-                    "--tenant-id",
-                    str(tenant_id),
-                    "--timeline-id",
-                    str(timeline_id),
-                    "--pg-version",
-                    self.env.pg_version,
-                ]
-            )
-        else:
-            res = self.raw_cli(
-                [
-                    "tenant",
-                    "create",
-                    "--tenant-id",
-                    str(tenant_id),
-                    "--timeline-id",
-                    str(timeline_id),
-                    "--pg-version",
-                    self.env.pg_version,
-                ]
-                + sum(list(map(lambda kv: (["-c", kv[0] + ":" + kv[1]]), conf.items())), [])
-            )
+
+        args = [
+            "tenant",
+            "create",
+            "--tenant-id",
+            str(tenant_id),
+            "--timeline-id",
+            str(timeline_id),
+            "--pg-version",
+            self.env.pg_version,
+        ]
+        if conf is not None:
+            args += sum(list(map(lambda kv: (["-c", kv[0] + ":" + kv[1]]), conf.items())), [])
```

Comment:
it's getting hard to read this "one-liner"

```python
from itertools import chain, product

args.extend(chain.from_iterable(product(["-c"], (f"{key}:{value}" for key, value in conf.items()))))
```

that one too, but it will not create new lists. However, I can propose

```python
from itertools import cycle

for arg in zip(cycle(["-c"]), (f"{key}:{value}" for key, value in conf.items())):
    args.extend(arg)
```

Reply:
I've used the former option, thanks for the suggestion.